apps/chat/channels.py

```diff
@@ -556,16 +557,24 @@ def _inform_user_of_error(self):
         """Simply tells the user that something went wrong to keep them in the loop. This method wil not raise an error
         if something went wrong during this operation.
         """
+
         try:
             bot_message = self._generate_response_for_user(
                 """
                 Tell the user that something went wrong while processing their message and that they should
                 try again later
                 """
             )
+        except Exception as e:
+            logger.exception(
+                f"Something went wrong while trying to generate an appropriate error message for the user\n\n{e}"
```

Comment:
nit: don't add the error to the message. The log record will include the exception details automatically and adding the error to the messages breaks exception grouping in tools like sentry

Reply:
Oh, I did not know that. https://github.com/dimagi/open-chat-studio/pull/1240/commits/76a11d6fba4004abc3074f1906116c76cb134867